Tests/Indicators/ImpliedVolatilityTests.cs

```diff
@@ -44,77 +46,138 @@ public void SetUp()
             DividendYieldUpdatesPerIteration = 1;
         }
 
-        // For comparing IB's value
-        [TestCase("SPX230811C04300000", 0.2)]
-        [TestCase("SPX230811C04500000")]
-        [TestCase("SPX230811C04700000")]
-        [TestCase("SPX230811P04300000", 0.015)]
-        [TestCase("SPX230811P04500000")]
-        [TestCase("SPX230811P04700000", 0.075)]
-        [TestCase("SPX230901C04300000")]
-        [TestCase("SPX230901C04500000")]
-        [TestCase("SPX230901C04700000")]
-        [TestCase("SPX230901P04300000")]
-        [TestCase("SPX230901P04500000")]
-        [TestCase("SPX230901P04700000")]
-        [TestCase("SPY230811C00430000", 0.055)]
-        [TestCase("SPY230811C00450000", 0.015)]
-        [TestCase("SPY230811C00470000")]
-        [TestCase("SPY230811P00430000", 0.015)]
-        [TestCase("SPY230811P00450000")]
-        [TestCase("SPY230811P00470000", 0.075)]
-        [TestCase("SPY230901C00430000", 0.025)]
-        [TestCase("SPY230901C00450000")]
-        [TestCase("SPY230901C00470000")]
-        [TestCase("SPY230901P00430000")]
-        [TestCase("SPY230901P00450000")]
-        [TestCase("SPY230901P00470000", 0.035)]
```

Comment:
Just double checking here but the error margin seems to have gone up, compared with the previous version 3% vs 8%?

Reply:
The previous 3% is absolute error (e.g. 15%IV +/- 3%), now it is relative error (e.g. 15%IV +/- 1.2%)